### PR TITLE
Hardcoded in a fix for the parargraph placement

### PR DESCRIPTION
**Pull request by edward-bian**
Program now runs in full and prints out the full chapter.

### CODE REVIEW

**Comment by physikerwelt** [comment]
Niche change in general. But the comments go above the code. I got the impression that you put them after the code.

File: KLSadd_insertion/updateChapters.py [hunk 1]
```diff
@@ -86,11 +81,13 @@ def findReferences(chapter):
     references = []
     index = -1
     chaptercheck = 0
+    #chaptercheck designates which chapter is being searched for references
```
Comment (physikerwelt) [inline KLSadd_insertion/updateChapters.py]: This should read
```
     #chaptercheck designates which chapter is being searched for references
      chaptercheck = 0
```

**Comment by physikerwelt** [comment]
ping @edward-bian @HowardCohl any updates here?

**Comment by physikerwelt** [comment]
@HowardCohl did you hear back from @edward-bian ?

**Comment by edward-bian** [comment]
The new branch should have moved all of the comments into the right places.

**Comment by HowardCohl** [comment]
@edward-bian @physikerwelt Nice. Thanks!

**Pull request merged**